test/test_core.py

```diff
@@ -9749,6 +9749,19 @@ def test_externref(self):
     self.emcc_args += ['-mreference-types']
     self.do_core_test('test_externref.c', libraries=['asm.o'])
 
+  @parameterized({
+    '': [False],
+    'dynlink': [True]
+  })
+  @requires_node
+  @no_wasm2js('wasm2js does not support reference types')
+  def test_externref2(self, dynlink):
```

Comment:
Can we give this test a better name.  How about `test_externref_em_js` ?  (and name the source file likewise).

Reply:
done.